src/test/java/com/carmen/app/control/CarServiceTest.java

```diff
@@ -41,16 +43,20 @@
 	private TypedQuery<Car> query;
 	
 	private Car car;
+	
+
 
 	private String id = "dff6b9ba-8579-11eb-8dcd-0242ac130003";
 
 	@Before
 	public void setUp() throws Exception {
```

Comment:
If car is modified in one test, it would affect any other test. Unit tests should be totally independent. car should have local scope for every test